### PR TITLE
Build c2cgeoportal using Docker multistage build

### PR DESCRIPTION
**Pull request by sbrunner**
=> Remove the docker-run and docker-compose-run, move to Circle CI
Update the documentation
Update C2cupgrade messages
Update Docker containers diagrams
Reactivate call to updatepo
Reactivate clean DockerHub tags

I steel fave to fix the deployments

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,179 @@
+FROM camptocamp/c2cwsgiutils:2 AS base
```
Comment (pvalsecc) [inline Dockerfile]: Can't you upgrade to v3?

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+  typedoc typescript \
+  --src=/usr/lib/node_modules/ngeo/package.json --src=package.json --dst=npm-packages
+
+COPY admin/package.json /app/c2cgeoportal/admin/
+WORKDIR /app/c2cgeoportal/admin
+RUN \
+  npm --no-optional --no-package-lock install && \
+  npm cache clear --force && \
+  rm -rf /tmp/*
+
```
Comment (pvalsecc) [inline Dockerfile]: Missing splitting comment.

File: Dockerfile (hunk 1 above)
Comment (sbrunner) [reply Dockerfile]: +1 in another pull request :-)

File: Dockerfile (hunk 2 above)
Comment (sbrunner) [reply Dockerfile]: Added, thanks :-)

**Pull request merged**